jdaviz/configs/imviz/helper.py

```diff
@@ -146,8 +147,8 @@ def load_data(self, data, data_label=None, show_in_viewer=True, **kwargs):
         """
         prev_data_labels = self.app.data_collection.labels
 
-        if 'gwcs_to_fits_sip' not in kwargs and 'Orientation' in self.plugins.keys():
-            kwargs['gwcs_to_fits_sip'] = self.plugins['Orientation'].gwcs_to_fits_sip
+        # to pass to parser
+        kwargs['gwcs_to_fits_sip'] = gwcs_to_fits_sip
```

Comment:
I think this is a good intermediate solution.  @javerbukh - you should be able to just pass this directly as `self.load(..., gwcs_to_fits_sip=gwcs_to_fits_sip)`